Tasks: Restart cscreend server after config deployment (#266)
openSUSE/orthos2: docs/adminguide/setup.rst

        systemctl start orthos2.socket
        systemctl start orthos2.service
        systemctl start orthos2_taskmanager

Optional: Setup cscreen server
##############################

The Domain feature of "cscreen" server (`github.com/openSUSE/cscreen <https://github.com/openSUSE/cscreen>`_) requires

> the manual setup of a cscreen server. Do the following things to ensure that Orthos can regenerate the configuration:

Reviewer: ```suggestion
the manual setup of a cscreen server. Follow these steps to ensure that Orthos can regenerate the configuration:
```
